### DISCUSSION TITLE
Flexible packges

### DISCUSSION THREAD

**Discussion by jmauzyk** [dvdoug/BoxPacker#252] [Q&A] [answered]
Wondering if you have any recommendations / best practices for how to handle flexible packages like bubble mailers, since the outer/inner dimensions change depending on what you manage to stuff into them. Thanks!

**Comment by dvdoug** [comment, marked as answer]
Hi

I'm afraid I don't - there is an open feature request (#141) that would cover this and contains some workarounds others have found useful.

As a feature it is definitely something I want to add though.
